leits/MeetingBar
# 🐛 fix(calendars): show read only calendars from delegateSources

- after initialising the eventstore with the delegateSources a list of all calendars are displayed

### Status
**IN DEVELOPMENT**

### Description
The change shows the readonly calendars from the delegates in the macos calendar as well.

### Steps to Test or Reproduce
- create a delegate in macos calendar
- check the calendar tab in meetingbar that the calendar shows up
